packages/components/src/components/Card/Card.stories.tsx

```diff
@@ -35,12 +35,24 @@ const body = (
   </Paragraph>
 );
 
-const contentContainerStyle = {
-  padding: 'var(--dds-spacing-x0-75)',
-};
+const contentContainerStyle = (
+  <style>
+    {`
+.story-container-padding {
+  padding: var(--dds-spacing-x0-75);
+}
+  `}
+  </style>
+);
 
 export const Default: Story = {
-  args: { children: <div style={contentContainerStyle}>Content</div> },
+  args: {
+    children: (
+      <div className="story-container-padding">
+        Content {contentContainerStyle}
+      </div>
+    ),
+  },
```

Comment:
Legge styling inn i decorators kanskje? Da dukker de ikke opp i kodeblokkene i dokumentasjon.
```suggestion
  decorators: [
    (Story) => (
      <>
        <style>
          {`
          .story-container-padding {
            padding: var(--dds-spacing-x0-75);
          }
          `}
        </style>
        <Story />
      </>
    )
  ],
  args: {
    children: (
      <div className="story-container-padding">
        Content
      </div>
    ),
  },
```

Reply:
Ser den, ja. Hadde lyst på en løsning som gjør det ryddig for både konsumenter og vedlikehold. Jeg vil gjerne bruke styling via klasser, slik at jeg ser hvordan det oppfører seg i forhold til styling fra komponenten, og inline har for høy precedence. Kom ikke på en god måte via en CSS-fil, men decorators kan jeg prøve 👍

Reply:
Hva med nå?

Reply:
Det ble ryddig! 🧹 Nå blir kodeblokkene i dokumentasjon rene og pene ☺️ 
![image](https://github.com/user-attachments/assets/a2a2b249-b54b-4923-b073-0c34bb66ea7e)